openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -748,17 +752,16 @@ Authorization: BEARER czZCaGRSa3F0MzpnWDFmQmF0M2JW
 
 {
    "credential_identifier": "CivilEngineeringDegree-2023",
-   "proof": {
-      "proof_type": "jwt",
-      "jwt":
-      "eyJ0eXAiOiJvcGVuaWQ0dmNpLXByb29mK2p3dCIsImFsZyI6IkVTMjU2IiwiandrI
-      jp7Imt0eSI6IkVDIiwiY3J2IjoiUC0yNTYiLCJ4IjoiblVXQW9BdjNYWml0aDhFN2k
-      xOU9kYXhPTFlGT3dNLVoyRXVNMDJUaXJUNCIsInkiOiJIc2tIVThCalVpMVU5WHFpN
-      1N3bWo4Z3dBS18weGtjRGpFV183MVNvc0VZIn19.eyJhdWQiOiJodHRwczovL2NyZW
-      RlbnRpYWwtaXNzdWVyLmV4YW1wbGUuY29tIiwiaWF0IjoxNzAxOTYwNDQ0LCJub25j
-      ZSI6IkxhclJHU2JtVVBZdFJZTzZCUTR5bjgifQ.-a3EDsxClUB4O3LeDD5DVGEnNMT
-      01FCQW4P6-2-BNBqc_Zxf0Qw4CWayLEpqkAomlkLb9zioZoipdP-jvh1WlA"
-   }
+   "proofs": [
+      {
+        "proof_type": "jwt",
+        "jwt": "eyJ0eXAiOiJvcGVuaWQ0dmNpLXBy...mlkLb9zioZoipdP-jvh1WlA"
+      },
+      {
+        "proof_type": "jwt",
+        "jwt": "eyJ0eXAiOiJvcGVuaWQ0dmNpLXBy...mlkLb9zioZoipdP-jvh1WlA"
+      }
+   ]
```

Comment:
```suggestion
   "proofs": {
      "jwt": [
         "eyJ0eXAiOiJvcGVuaWQ0dmNpL...Lb9zioZoipdP-jvh1WlA",
         "eyJraWQiOiJkaWQ6ZXhhbXBsZ...KPxgihac0aW9EkL1nOzM"
      [
   }
```

Reply:
This seems better.

Reply:
This makes sense if we assume a credential data set always maps to a single credential format. And I think we do. However, an example for such a multi format data set would be a PID (Person Identification Data under eIDAS) with instances in mdoc and sd-jwt vc. I just realize that the differentiation in separate data sets may have an impact on the UX as the user consents on the level of the data set, right?